docs/api/specs/webConnectionProtocol.md

```diff
@@ -0,0 +1,382 @@
+---
+id: webConnectionProtocol
+sidebar_label: Web Connection Protocol
+title: FDC3 Web Connection Protocol (next)
+---
+
+:::info _[@experimental](../fdc3-compliance#experimental-features)_
+
+The FDC3 Web Connection Protocol (WCP) is an experimental feature added to FDC3 in 2.2. Limited aspects of its  design may change in future versions and it is exempted from the FDC3 Standard's normal versioning and deprecation polices in order to facilitate any necessary change.
```

Comment:
Minor point - In this info box we have the same "FDC3" qualifier consistency issue as with the title (see above comment) between this page and the equivalent info box in desktopAgentCommunicationProtocol.md

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85